config/vufind/RateLimiter.yaml

```
@@ -40,6 +40,13 @@ Storage:
     #redis_version            : 6
     #redis_standalone         : true
 
+  turnstileOptions:
+    # Namespace for Turnstile result cache (default is Turnstile)
+    # namespace: Turnstile
```

We typically do not use spaced comments on actual settings, to help differentiate them from descriptive comments.

```suggestion
    #namespace: Turnstile
```